src/soroban/server.ts

```diff
@@ -307,14 +307,14 @@ export class Server {
     hash: string
   ): Promise<Api.GetTransactionResponse> {
     return this._getTransaction(hash).then((raw) => {
-      let successInfo: Omit<
+      let foundInfo: Omit<
```

Comment:
it won't happen each time, it'll just happen once at "compile time" when the TypeScript is turned into JavaScript. It's not part of the API because this is specifying "the subset of the 'success/failure' schema that does not contain the 'missing' schema," which isn't like a well-defined API concept, it's just useful here because we're trying to create a partially-(but well-)defined subset of an object.